### DISCUSSION TITLE
SO 2.3.10 after upgrade to 2.3.90 no endpoints logs, no network packets shown :-(

### DISCUSSION THREAD

**Discussion by UMWP** [Security-Onion-Solutions/securityonion#6459] [Unsupported Versions] [answered]
Hello.

Securityonion-2.3.10.iso installed in standalone mode and correctly configured. Windows logs from winlogbeat and sysmon received. Network connections also visible. After upgrade by sudo soup command - soup updated itself and sudo soup done again. Upgrade to version 2.3.90 took a while but finished successfully. After reboot and waiting for couple minutes all services are up and running (sudo so-status). Anyway I used salt-call state.highstate command to be sure, but no errors occured.

Host logs and network connections still received - checked by tcpdump on SO (for example on tcp port 5044) but not shown in kibana  last logs graph on the right (logs over time panel). Last logs seen before reboot time.

Latest Kibana log lines (no new lines - last time stamp before reboot):
{"ecs":{"version":"1.9.0"},"@timestamp":"2021-12-06T11:25:17.238+00:00","message":"Enabling the Chromium sandbox provides an additional layer of protection.","log":{"level":"WARN","logger":"plugins.reporting"},"process":{"pid":293}}
{"ecs":{"version":"1.9.0"},"@timestamp":"2021-12-06T11:25:20.926+00:00","message":"Dependent plugin setup complete - Starting ManifestTask","log":{"level":"INFO","logger":"plugins.securitySolution"},"process":{"pid":293}}
{"ecs":{"version":"1.9.0"},"@timestamp":"2021-12-06T11:25:30.212+00:00","message":"Kibana is now available (was degraded)","log":{"level":"INFO","logger":"status"},"process":{"pid":293}}

Latest elasticsearch log:
[2021-12-06T13:21:55,849][DEPRECATION][org.elasticsearch.deprecation.cluster.metadata.IndexNameExpressionResolver] this request accesses system indices: [.apm-agent-configuration, .apm-custom-link, .async-search, .kibana_1, .kibana_7.15.2_001, .kibana_task_manager_7.15.2_001, .security-7, .tasks], but in a future major version, direct access to system indices will be prevented by default

Last suricata log lines (no new lines - last time stamp before reboot)::
ules, 3982 are inspecting packet payload, 18494 inspect application layer, 0 are decoder event only
6/12/2021 -- 11:23:49 - <Info> - Going to use 1 thread(s)
6/12/2021 -- 11:23:49 - <Info> - Running in live mode, activating unix socket
6/12/2021 -- 11:23:49 - <Info> - Using unix socket file '/var/run/suricata/suricata-command.socket'
6/12/2021 -- 11:23:49 - <Notice> - all 1 packet processing threads, 4 management threads initialized, engine started.
6/12/2021 -- 11:23:49 - <Info> - All AFP capture threads are running.

Appreciate any help.
Regards.

**Comment by Xboarder56** [comment]
I have this same issue as well but upgrading from 2.80 to 2.90. TCP dump has logs, so status shows everything as good but nothing in SOC or kibana for logs including both zeek logs and syslog.

Edit: didn't mean to send this in as an answer just wanted to provide additional info as I have the same problem.

**Comment by defensivedepth** [comment, marked as answer]
@UMWP Can you check your disk space on both root and /nsm?

What is the output of the following? `sudo so-redis-count`

Can you also check Grafana graphs and see if anything looks abnormal?

@Xboarder56 Per the guidelines here (https://github.com/Security-Onion-Solutions/securityonion/discussions/1720), please create a new issue.

**Reply by UMWP** [reply]
I think we can close this topic - I have dismissed 2.3.10 version upgrade and started from fresh 2.3.90-WAZUH.iso with winlogbeat-oss-7.15.2-windows-x86_64.msi installer downloaded from it. This is first/test SO deployment so we can experiment with that.

Thanks for your help
